### ISSUE TITLE
"requireTrailingComma: true" + "disallowTrailingComma: true" + autofix toggles comma

### ISSUE THREAD

**Issue by jamietre** [jscs-dev/node-jscs#1905]
test.js:

    var x = {
        a: 1,
        b: 2,
    };

.jscsrc

    {
        "disallowTrailingComma": true
    }

Then

    > jscs test.js --fix

(comma is removed)

    > jscs test.js --fix

(comma is added again)

**Comment by markelog** [comment]
Can't repro, @hzoo could confirm?

**Comment by hzoo** [comment]
Yeah I can't repro either actually. @jamietre Can you check again? And what version are you using?

**Comment by jamietre** [comment]
Interesting.. I even set up the simple scenario just as in this report to
verify it. Fresh install from npm public registry yesterday. I am on
windows in full disclosure. Will try it in ubuntu vm today and see if I can
repro shortly.

**Comment by jamietre** [comment]
Here's the issue... I didn't actually use that `.jscsrc`, this should repro it:

    {
        "preset": "node-style-guide",
        "validateLineBreaks": false,
        "disallowTrailingComma": true,
    }

It seems like there's a conflct between the `requireTrailingComma` directive from the preset. If you add `"requireTrailingComma": false` then it works. 

Not sure if this is a general problem or specific to this rule, but it seems like any `disallow` rule should invalidate a previous `require` rule and vice-versa.

**Comment by hzoo** [comment]
I think we like a lot of projects would want to keep rules mostly independent of each other.
Although with 3.0 we should combine require/disallow like in https://github.com/jscs-dev/node-jscs/issues/698 so this won't be an issue.

It's a general issue but the point is that normally you disable the rules in a preset so I would do `"requireTrailingComma": false` like you mentioned. http://jscs.info/overview#preset
Basically when using a preset you would first run the preset, and disable the rules in the preset and then add the required rules (at least for now)

**Comment by lf94** [comment]
If this could be added to the "disableTrailingComma" documentation that would be fantastic. I wasted an hour trying to figure out what was going on.

**Comment by markelog** [comment]
@lf94 do you wanna do it?
